src/hooks/Mock/useFetchMock.ts

```diff
@@ -0,0 +1,38 @@
+export const useFetchMock = () => {
```

Comment:
Why is this a hook if does not have interaction with state?

Reply:
I made that hook just as an example, I forgot to delete it now I'll do it.